edb/lang/graphql/types.py

```diff
@@ -468,8 +474,11 @@ def __init__(self, schema, **kwargs):
         if edb_base is None and self.edb_type:
             edb_base = schema.edb_schema.get(self.edb_type)
 
-        assert edb_base is not None
-        assert schema is not None
+        if edb_base is None:
+            raise GraphQLTranslationError(
+                "cannot determine the EdgeDB base type")
```

Comment:
`f"cannot determine the EdgeDB base type for {self.edb_type.__name__}"`

^ do we have a unittest for this?

Reply:
Good point, let me see if I can still reach this exception. It may be a relic of early versions.

Reply:
Looks like `schema.get()` cannot return `None`.